src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- [ ] Nice for applying the change. I would advise you not to disable the Linter. The general requirement says that there must not be linters errors:
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).


- [ ]  There are a few reasons for using linters:


- Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.

Reply:
Hello @OLIPLICHE thank you for the feedback, however based on official [Redux documentation](https://redux-toolkit.js.org/api/createSlice) I am creating and using createSlice the intended way. The problem is that esLint doesn't yet recognize this new way of working with createSlice in Redux, therefore it marks it as incorrect and throws this linter error. I've discussed this with other programmers in the Microverse program, and they've all concurred that the way I wrote my code is the most efficient way of writing createSlice functions without reverting to previous more complex methods. I hope that they'll eventually patch this conflict and that you understand that I'm not trying to cheat the activity by using no-param-reassign on this file, I'm just implementing the newest recommended syntax. If you have any doubts please let me know.